globalTypes.d.lua

```diff
@@ -0,0 +1,10216 @@
+
```

Comment:
Is this StyLua-generated? Does it need to be committed?

Reply:
No. Committing this was a mistake and I have no idea how I didn't notice.

Reply:
Removed and added relevant files to gitignore